examples/office/importers/utrade/tests/expected/d60c05338c9bb13c44249d0c9adefdde4ea273a5.beancount

```diff
@@ -1,24 +1,25 @@
+;;; document filing path: Assets/US/UTrade/2015-02-25-utrade.UTrade20150225.csv
```

Comment:
Maybe choose something a bit more "programmatic" than "document filing path".
Maybe: "file_name", matching the method name,  so it's more explicit?

Also, you could contemplate using  Custom directive. Your call, don't  care that much.

Reply:
See above, this is really the relative path to which the document would be filed.

Reply:
Also, I thought about using a `custom` or `note` directive, but then I would need to come up with a date for the directive, and I could not think to any that made sense.